multi-node-test.sh

```diff
@@ -148,114 +107,114 @@ df -h" 2>&1 | dshbak -c; then
     exit 1
 fi
 
+#if ! ssh -i ci_key jenkins@"${nodes[0]}" "set -x
+#set -e
+#sudo mkdir -p $CART_BASE
+#sudo mount -t nfs $HOSTNAME:$PWD $CART_BASE"; then
+#    rc=${PIPESTATUS[0]}
+#fi
+
+# put yaml files back
+restore_dist_files() {
+    local dist_files="$*"
+
+    for file in $dist_files; do
+        if [ -f "$file".dist ]; then
+            mv -f "$file".dist "$file"
+        fi
+    done
+
+}
+
+TEST_TAG="${3:-quick}"
+
+TESTDIR=${SL_PREFIX}/TESTING
+
+LOGDIR="$TESTDIR/avocado/job-results/CART_${2}node"
+
+echo SCHAN15 - PWD $PWD
+echo SCHAN15 - HOST `hostname`
+echo SCHAN15 - LS `ls install`
+echo SCHAN15 - LS `ls install/Linux`
+echo SCHAN15 - LS `ls install/Linux/TESTING`
+echo SCHAN15 - LS `ls install/Linux/TESTING/rpc`
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of deprecated `..` [SC2006]